pkg/config/application.go

```diff
@@ -24,36 +24,36 @@ type Application struct {
 	ACLs          map[string]string
 }
 
-// GetApplicationConfiguration returns the existing application configuration values from a config
+// ApplicationConfiguration returns the existing application configuration values from a config
 // transaction as an Application type. This can be used to retrieve existing values for the application
 // prior to updating the application configuration.
-func (c *ConfigTx) GetApplicationConfiguration() (Application, error) {
+func (c *ConfigTx) ApplicationConfiguration() (Application, error) {
 	applicationGroup, ok := c.base.ChannelGroup.Groups[ApplicationGroupKey]
 	if !ok {
 		return Application{}, errors.New("config does not contain application group")
 	}
 
 	var applicationOrgs []Organization
 	for orgName := range applicationGroup.Groups {
-		orgConfig, err := c.GetApplicationOrg(orgName)
+		orgConfig, err := c.ApplicationOrg(orgName)
 		if err != nil {
 			return Application{}, fmt.Errorf("retrieving application org %s: %v", orgName, err)
 		}
 
 		applicationOrgs = append(applicationOrgs, orgConfig)
 	}
 
-	capabilities, err := c.GetApplicationCapabilities()
+	capabilities, err := c.ApplicationCapabilities()
 	if err != nil {
 		return Application{}, fmt.Errorf("retrieving application capabilities: %v", err)
 	}
 
-	policies, err := c.GetPoliciesForApplication()
+	policies, err := c.PoliciesForApplication()
```

Comment:
We may want to consider renaming this to `ApplicationPolicies` for consistency with the other retrievers.